portality/formcontext/forms.py

```diff
@@ -373,13 +385,13 @@ class PublicApplicationForm(JournalInformation, Suggestion, PublicSuggester):
     pass
 
 
-class ManEdApplicationReviewForm(Editorial, Workflow, ApplicationOwner, JournalInformation, Suggestion, Subject, AdminSuggester, Notes):
+class ManEdApplicationReviewForm(Editorial, Workflow, ApplicationOwner, JournalInfoOptionalPaymentURLs, Suggestion, Subject, AdminSuggester, Notes):
     """
     Managing Editor's Application Review form.  It consists of:
         * Editorial - ability to add editorial groups (but ability to add editors individually will be disabled)
         * Workflow - ability to change application status
         * ApplicationOwner - able to set the owner, and for the owner assignment to be optional except under certain circumstances
-        * JournalInformation - journal bibliographic data
+        * ManEdJournalInfoOverride - journal bibliographic data, with custom validation for managing editors
```

Comment:
the name's changed to JournalInfoOptionalPaymentURLs, not ManEdJournalInfoOverride. I can fix it in the hotfix branch if you like.